charts/cloudzero-agent/templates/check-cm.yaml

```diff
@@ -0,0 +1,105 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: validator
+data:
+  script.sh: |
+    set -e;
+    echo 'Validation starting...';
+    python -m venv /app/venv;
```

Comment:
I may be missing something, but is the intent to activate the virtual environment?

Reply:
Yes - basically. 🙈  This is me forgetting to remove this line. Which should be part of the packaging task I have to do next. I can remove this for now.